app/components/billing/subscription-manager.tsx

```
@@ -21,9 +21,9 @@ export function SubscriptionManager() {
 
   useEffect(() => {
     loadSubscriptionData();
-  }, []);
+  }, [loadSubscriptionData]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)